src/js/components/agencyV2/AgencyOverview.jsx

```diff
@@ -0,0 +1,121 @@
+/**
+ * AgencyOverview.jsx
+ * Created by Lizzie Salita 3/16/21
+ */
+
+import React, { useState, useEffect } from 'react';
+import { useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { throttle } from 'lodash';
+
+import { mediumScreen } from 'dataMapping/shared/mobileBreakpoints';
+import ExternalLink from 'components/sharedComponents/ExternalLink';
+import ReadMore from 'components/sharedComponents/ReadMore';
+
+const AgencyOverview = () => {
+    const {
+        name,
+        logo,
+        website,
+        mission,
+        congressionalJustification,
+        showAboutData,
+        subtierCount
+    } = useSelector((state) => state.agencyV2.overview);
+
+    const [windowWidth, setWindowWidth] = useState(window.innerWidth);
+
+    useEffect(() => {
+        const handleResize = throttle(() => {
+            setWindowWidth(window.innerWidth);
+        });
+        window.addEventListener('resize', handleResize);
+        return () => window.removeEventListener('resize', handleResize);
+    }, []);
+
+    const image = logo ? (
+        <img
+            className="agency-overview__image"
+            src={`graphics/agency/${logo}`}
+            alt={`${name} logo`} />
+    ) : '';
+
+    const missionBlock = (
+        <div className="agency-overview__data">
+            <h4>Agency Mission</h4>
+            <p>{mission || '--'}</p>
+        </div>
+    );
+
+    const aboutBlock = (
+        <div className="agency-overview__data">
+            <h4>About this Agency&apos;s Data</h4>
+            <p>
+                There is a 90 day delay in displaying contract award data, subcontract data,
+                and Account Breakdown by Award (File C) data for the Department of Defense (DOD).
+                For more information, visit our <Link to="/about?section=data-quality">About Page</Link>.
+                To see a complete list of this agency&apos;s submissions, visit our&nbsp;
+                <Link to="/submission-statistics/agency/097">Submission Statistics page</Link>.
+            </p>
+        </div>
+    );
+
+    const websiteBlock = (
+        <div className="agency-overview__data">
+            <h4>Website</h4>
+            {website ? <ExternalLink url={website} /> : '--'}
+        </div>
+    );
+
+    const cjBlock = (
+        <div className="agency-overview__data">
+            <h4>Congressional Justification of Budget (CJ)</h4>
+            {congressionalJustification ? <ExternalLink url={congressionalJustification} /> : '--'}
```

Comment:
Because these <ExternalLink>s render as buttons, the address doesn't show in the browser status bar. Also this may have screen-reader implications (not standard for <Button> to navigate). I don't plan to request this PR is changed, just considering...

Reply:
Scratch, now I see they open a modal with A links...